source/contracts/reporting/Market.sol

```diff
@@ -324,8 +331,21 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         require(_forkingMarket != this);
         require(!isFinalized());
         IInitialReporter _initialParticipant = getInitialReporter();
+        // Early out if already disavowed or nothing to disavow
+        if (_initialParticipant.getReportTimestamp() == 0) {
+            return true;
+        }
         delete participants;
         participants.push(_initialParticipant);
+        // Send REP from the no show bond back to the address that placed it. If a report has been made tell the InitialReporter to return that REP and reset
+        IReputationToken _reputationToken = getReputationToken();
+        uint256 _balance = _reputationToken.balanceOf(this);
```

Comment:
If someone transfers 1 attorep to the market after initial report, it would prevent the original inital reporter from getting their rep back? And whoever performed the migrateThroughOneFork would get all the original reporter's staked rep?